rules/properties-alphabetical-order/checkNode.js

```diff
@@ -3,11 +3,16 @@ import { checkAlphabeticalOrder } from '../checkAlphabeticalOrder.js';
 import { isCustomProperty } from '../../utils/isCustomProperty.js';
 import { isStandardSyntaxProperty } from '../../utils/isStandardSyntaxProperty.js';
 import * as vendor from '../../utils/vendor.js';
+import sortNodeProperties from 'postcss-sorting/lib/properties-order/sortNodeProperties.js';
 
 // eslint-disable-next-line max-params
-export function checkNode(node, result, ruleName, messages) {
+export function checkNode(node, result, ruleName, messages, isFixEnabled) {
 	let allPropData = [];
 
+	if (isFixEnabled) {
+		sortNodeProperties(node, { order: 'alphabetical' });
```

Comment:
This mixes what `checkNode` function suppose to do. It should only check and report violations. No changing of the code.